javascript/javascript2/week3/game/game.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+
+// counter for L
+let totalL = 0;
+const elL = document.querySelector("#countL");
+const pressedL = (e) => {
+  if (e.keyCode == 76) {
+    totalL++;
+    elL.innerHTML = totalL;
+  }
+};
+function counterL() {
+  document.addEventListener("keydown", pressedL);
+}
+// counter for S
+
+let totalS = 0;
+const elS = document.querySelector("#countS");
+const pressedS = (e) => {
+  if (e.keyCode == 83) {
+    totalS++;
+    elS.innerHTML = totalS;
+  }
+};
+function counterS() {
+  document.addEventListener("keydown", pressedS);
+}
+
+// setting the time of the game
+const startButton = document
+  .getElementById("timeButton")
+  .addEventListener("click", startGame);
+function startGame() {
+  const timeSet = document.getElementById("playTimeSet").value;
+  const label = document.getElementById("label");
+  if (timeSet < 1) {
+    label.innerHTML = "Please set the time!";
+    return;
+  } else {
+    label.innerHTML = "Game started!";
+  }
+  counterL();
+  counterS();
+  totalS = 0;
+  totalL = 0;
+  setTimeout(() => {
+    console.log("game finished");
+    if (totalS > totalL) {
+      label.innerHTML = "S is the WINNER!";
+    } else if (totalL > totalS) {
+      label.innerHTML = "L is the WINNER!";
+    } else {
+      label.innerHTML = "TIE!";
+    }
+    document.removeEventListener("keydown", pressedL);
+    document.removeEventListener("keydown", pressedS);
+  }, timeSet * 1000);
+}
```

Comment:
I tried to add a restart button to your game by using below code but...If You dont want just delete this comment enqira...
const btn2= document.getElementById("restart"); // added restart button in html
 /* here time,and label i made them global..;const time = document.getElementById("playTimeSet");
const label = document.getElementById("label");
and also i made a change  in this function..
function startGame() {
 const timeSet = time.value;
*/

function restartGame(){
  time.value = "";
label.textContent = "Play again";
totalS= totalL=0;
elS.textContent = "";
elL.textContent = "";
}
btn2.addEventListener("click", restartGame)*/